newIDE/app/public/external/piskel/piskel-electron-index.html

```diff
@@ -0,0 +1,17 @@
+<html>
+  <head>
+    <title>GDevelop Pixel Editor (Piskel)</title>
```

Comment:
should this have a more generic name, like GDevelop image editor?

Reply:
Done